client/question/domain/CodeEvalResultObjectFactorySpec.js

```diff
@@ -607,6 +607,18 @@ describe('CodeEvalResultObjectFactory', function() {
     });
   });
 
+  describe('getErrorLineNumber', function() {
+    it('should return correct error line number', function() {
+      var someErrorTraceback = ErrorTracebackObjectFactory.create(
+        'Error on line 5', [TracebackCoordinatesObjectFactory.create(5, 1)]);
```

Comment:
Typically the error message does not have a line number. If that's the case we should not include one in the first argument of the call to ErrorTracebackObjectFactory.create(). 

Ditto below.